.github/workflows/pull_request.yml

```diff
@@ -14,7 +14,7 @@ jobs:
           fetch-depth: 1
 
       - name: Static Analysis
-        run: ./gradlew ktlintCheck app:lintDebug core:lintDebug custom:lintCustomexampleDebug
+        run: ./gradlew ktlintCheck app:lintDebug custom:lintCustomexampleDebug
```

Comment:
Why has this changed?

Reply:
We are now linting core as part of app by using `lintOptions.isCheckDependencies=true`

This gives us a more accurate lint report. Core contains strings only used in App so when you lint core by itself it says "unused", the ideal solution would be to migrate the strings+translations to their appropriate module but that is not possible at the moment, I've created a ticket #1788 to get the end result we want